core/domain/config_domain.py

```diff
@@ -222,3 +222,25 @@ def update_moderator_ids():
     SET_OF_STRINGS_SCHEMA,
     'Banned usernames (editing permissions for these users have been removed)',
     [])
+
+
+# TODO(milit): Check the language codes are valid
+LANGUAGES_SUPPORTED = ConfigProperty(
+    'languages_supported',
+    {
+        'type': 'list',
+        'items': {
+            'type': 'dict',
+            'properties': [{
+                'name': 'name',
+                'description': 'Name of the language',
+                'schema': {'type': 'unicode'},
+            }, {
+                'name': 'code',
+                'description': 'The code of the language in the translations',
+                'schema': {'type': 'unicode'},
+            }]
+        }
+    },
+    'Languages supported by the platform.',
+    [{'name': 'English', 'code': 'en'}, {'name': u'Español', 'code': 'es'}])
```

Comment:
Add
```
  # coding: utf-8
  #
```
to the top of this file, otherwise Python can't handle the non-ascii chars.